README.md

```diff
@@ -13,21 +13,28 @@ you can create complex workflows with very little code.
 ## Installation
 
 The easiest way to install the Fixie SDK is via `pip`:
+
 ```
-pip install fixie
+pip install fixieai
 ```
 
-## Get your API key
+## Build Agents
+
+Agents are at the heart of the Fixie ecosystem, and we make it easy to build and contribute your own. To get started, check out our [examples repo](https://github.com/fixie-ai/fixie-examples) or [read the docs](https://docs.fixie.ai/agents/).
+
+## Using the Fixie CLI
 
-You can get your Fixie API key on your profile page at https://app.fixie.ai.
+In order to get started making requests to Fixie, you'll need an API key. You can get your Fixie API key on your profile page at https://app.fixie.ai.
```

Comment:
In order to get started making requests to Fixie, you'll need to authenticate. Simply:
```bash
fixie auth
```